libcontainer/container_linux.go

```diff
@@ -195,16 +191,33 @@ func (c *linuxContainer) Run(process *Process) error {
 	if err != nil {
 		return err
 	}
-	isInit := status == Stopped
-	if err := c.start(process, isInit); err != nil {
+	if err := c.start(process, status == Stopped); err != nil {
 		return err
 	}
-	if isInit {
-		return process.ops.signal(InitContinueSignal)
+	if status == Stopped {
+		return c.Exec()
 	}
 	return nil
 }
 
+func (c *linuxContainer) Exec() error {
+	path := filepath.Join(c.root, execFifoFilename)
+	f, err := os.OpenFile(path, os.O_RDONLY, 0)
```

Comment:
True but we don't support multiple processes handling the container.  I can try to protect the libcontainer call with a lock but its not going to help out of process calls.